string token
string timestamp
float64[] ego2global
int32[] ids
float64[] obs
float64[] preds
float64[] scores
int32 pred_num